packages/frontend/amp/components/lib/AMPAttributes.ts

```diff
@@ -8,6 +8,5 @@ declare module 'react' {
     interface HTMLAttributes<T> extends DOMAttributes<T> {
         on?: string;
         overflow?: string;
-        fallback?: string;
```

Comment:
As a fallback will only be used as a fallback, the type here could be `fallback: true` which means (with a tslint tweak) we could `<amp-fallback fallback>...` which still outputs `<amp-fallback fallback="true">...`